auto_dev/protocols/formatter.py

```diff
@@ -144,8 +168,12 @@ def encode_element(element) -> str:
     elements = filter(lambda e: not isinstance(e, (MessageAdapter, ast.Enum)), message.elements)
     inner = "\n".join(map(encode_element, elements))
     indented_inner = textwrap.indent(inner, "    ")
-    return f"@staticmethod\ndef encode(proto_obj, {message.name.lower()}: {message.name}) -> None:\n{indented_inner}"
-
+    return (
+        "@staticmethod\n"
+        f"def encode(proto_obj, {message.name.lower()}: {message.name}) -> None:\n"
+        f"    \"\"\"Encode {message.name} to protobuf.\"\"\"\n\n"
+        f"{indented_inner}\n"
+    )
```

Comment:
empty encoder (since it returns `None`) would crash for empty messages without a docstring. Plus linter would inevitably complain as well later on